core/asset/asset_test.go

```diff
@@ -269,6 +269,7 @@ func TestAssetPatch(t *testing.T) {
 		description   string
 		asset         asset.Asset
 		patchDataJSON json.RawMessage
+		patchData     map[string]interface{}
```

Comment:
Mas, having both of them is more convenient at the moment for testing `core/asset/patch.go`. Because it is mostly easy to pass patch data as jsonstring in most cases, and patchData when I wanted to pass values of `labels` and `owners` in the field as of different data types.

Reply:
@StewartJingga made these changes!!